### PR TITLE
Fix: Missing navigation bar ordering description (fixes #162)

### LINKED ISSUE #162: Missing navigation bar ordering description
### Subject of the issue/enhancement/features
The navigation bar order feature is missing description text to explain its purpose.

### Related Tickets
#61 

### Your environment
* FW v5.24.7
* AAT v1

### Steps to reproduce
Please see project settings in AAT v1.

### Expected behaviour
A tooltip descriptor should explain its purpose.

### PR DESCRIPTION
**Pull request by swashbuck**
Fixes #162 

### Fix
* Adds a description (tooltip) for the `_navOrder` property
* Adds a title for `_navOrder` in _properties.schema_
* Minor adjustments to `_drawerPosition` - Use 'description' instead of 'help' in _course.schema.json_, missing space, and move closer to 'title'

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: properties.schema [hunk 1]
```diff
@@ -39,6 +39,8 @@
     "_navOrder": {
       "type": "number",
       "required": true,
+      "title": "Navigation bar order",
+      "help": "Determines the order in which the page level progress is displayed in the navigation bar. Negative numbers (e.g. -100) are left-aligned. Positive numbers (e.g. 100) are right-aligned.",
```
Comment (kirsty-hames) [inline properties.schema:43]: 'help' needs replacing with 'description'

File: properties.schema [hunk 2]
```diff
@@ -47,7 +49,8 @@
       "type": "string",
       "required": true,
       "default": "auto",
-      "title":"Drawer position",
+      "title": "Drawer position",
+      "help": "Determines the starting position of the drawer to open.",
```
Comment (kirsty-hames) [inline properties.schema:53]: 'help' needs replacing with 'description'

**Review by kirsty-hames — CHANGES_REQUESTED** [review]

**Review by kirsty-hames — CHANGES_REQUESTED** [review]
I've left a couple of comments but it's a +1 from me thanks @swashbuck

**Review by kirsty-hames — APPROVED** [review]

**Pull request merged**